### ISSUE TITLE
Syntax Error: Won't start

### ISSUE THREAD

**Issue by jscottelblein** [Tautulli/Tautulli#420]
I just grabbed the latest version (first time trying PlexPy), unpacked it, double-clicked plexpy.py, and the command window just flashed for a second, so I opened up a command window and manually typed in plexpy.py, and I get the following. Anyone have any suggestions? I'm not really a Python guy myself.

```
G:\Program Files\Servers\Media Servers\PlexPy>plexpy.py
Traceback (most recent call last):
  File "G:\Program Files\Servers\Media Servers\PlexPy\PlexPy.py", line 25, in <module>
    from plexpy import webstart, logger, web_socket
  File "G:\Program Files\Servers\Media Servers\PlexPy\plexpy\__init__.py", line 201
    except OSError, e:
                  ^
SyntaxError: invalid syntax
```

**Comment by Hellowlol** [comment]
Python 3 isnt supported. Download the latest in the 2.7 series

**Comment by STaRDoGG** [comment]
Thanks, I'm not real familiar with the intricacies of Python; I have both P3 and P2.7 installed on this machine, is there a way to force PlexPy to use 2.7?

**Comment by JonnyWong16** [comment]
I think the easiest method is to create a shortcut to `PlexPy.py`. Create a new shortcut with the following:

Target:

```
C:\Python27\pythonw.exe C:\Path\to\PlexPy\PlexPy.py
```

Start in:

```
C:\Python27
```

This will also run PlexPy without the command window.

**Comment by jscottelblein** [comment]
Thanks guys! I got it working. I actually created a service for it using FireDaemon, using the full path to the older Python, like @JonnyWong16 pointed out.

Only problem I seem to have now is trying to get to the settings page. When I click the Settings link, this shows (screenshot):

http://i.imgur.com/85ic3Bp.png

**Comment by Hellowlol** [comment]
I have no idea why that failed but i think you can fix it by manually set the pms name in the config file

**Comment by JonnyWong16** [comment]
Yeah, I've never seen that error before. Do any of your settings have special characters (non-ASCII)?

**Comment by jscottelblein** [comment]
lol, leave it to me to get an unseen error the first time I use a program.

Off the top of my head, the only "odd character" in my settings would be a pipe sign, i.e. "MyServer | Plex".

Though I wonder if, for some reason, it's choking on a few odd characters I have for Plex Library names. I have names like:

☆ Photos
★ Movies
☢ Last Chance

Other than than, there doesn't seem to be a whole lot even in my config.ini file. _scratches head_

**Comment by JonnyWong16** [comment]
I just tested those and I can still load the settings page. Have you tried to just reinstall PlexPy?

**Comment by jscottelblein** [comment]
I haven't, but, all I did on the original install was unzip the package to a folder, then run PlexPy.py, lol.

I'll try deleting the folder and doing it all over again though.

Update:
Figured it out. Turns out that if you don't have CHANGELOG.md in the same folder, that will happen. I tend to like keeping my system clean, by deleting changelog files, unnecessary ReadMe.txt's, and things of that nature, so, I had deleted that file right off the bat.

Maybe add a file check handler to make sure it exists first, and alert the user if it doesn't, instead of having an error page show? :)

**Comment by JonnyWong16** [comment]
Thanks

Note to self: https://github.com/drzoidberg33/plexpy/blob/master/plexpy/versioncheck.py#L253
`return '<h4>No changelog data</h4>'`

**Comment by jscottelblein** [comment]
=)

Just my opinion, it might be more helpful to add a more descriptive message there, since I got it while just trying to click the "Settings" link. Something along the lines of:

`return '<h4>Changelog.md missing from the main folder. Cannot display your Settings.</h4>'`

Otherwise a non-coder/Regular Joe user might be like, "Dude, I clicked on Settings, not Change Log". lol

**Comment by JonnyWong16** [comment]
Yeah, I fixed it in bfad769.

The settings page will still open if the file is missing, but the message will show up if you click on "Changelog".
